cmd/precise-code-intel-bundle-manager/internal/database/database_test.go

```diff
@@ -0,0 +1,269 @@
+package database
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sourcegraph/sourcegraph/cmd/precise-code-intel-bundle-manager/internal/types"
+	"github.com/sourcegraph/sourcegraph/internal/sqliteutil"
+)
+
+func init() {
+	sqliteutil.SetLocalLibpath()
+	sqliteutil.MustRegisterSqlite3WithPcre()
+}
+
+func TestDatabaseExists(t *testing.T) {
+	testCases := []struct {
+		path     string
+		expected bool
+	}{
+		{"cmd/lsif-go/main.go", true},
+		{"internal/index/indexer.go", true},
+		{"missing.go", false},
+	}
+
+	withTestDatabase(t, func(db *Database) {
+		for _, testCase := range testCases {
+			if exists, err := db.Exists(testCase.path); err != nil {
+				t.Errorf("unexpected error %s", err)
+			} else if exists != testCase.expected {
+				t.Errorf("unexpected exists result for %s. want=%v have=%v", testCase.path, testCase.expected, exists)
+			}
+		}
+	})
+}
+
+func TestDatabaseDefinitions(t *testing.T) {
+	// `\ts, err := indexer.Index()` -> `\t Index() (*Stats, error)`
+	//                      ^^^^^           ^^^^^
+
+	withTestDatabase(t, func(db *Database) {
+		if actual, err := db.Definitions("cmd/lsif-go/main.go", 110, 22); err != nil {
+			t.Errorf("unexpected error %s", err)
+		} else {
+			expected := []Location{
+				{
+					Path:  "internal/index/indexer.go",
+					Range: newRange(20, 1, 20, 6),
+				},
+			}
+
+			if diff := cmp.Diff(actual, expected); diff != "" {
+				t.Errorf("unexpected definitions locations (-want +got):\n%s", diff)
```

Comment:
I believe `-` is the first argument (i.e. `actual`), and `+` is the second argument (i.e. `expected`), so you actually want is 
```suggestion
			if diff := cmp.Diff(expected, actual); diff != "" {
				t.Errorf("unexpected definitions locations (-want +got):\n%s", diff)
```

(could help verify to make it fail on purpose)

Same for other places if I didn't get it wrong...

Reply:
Oops! Nice catch! Updated everywhere.